tests/Drupal/Commenting/TodoCommentUnitTest.php

```diff
@@ -38,7 +37,8 @@ protected function getErrorList(string $testFile): array
      */
     protected function getWarningList(string $testFile): array
     {
-        return [];
+        $errorList = (array_fill_keys(range(13, 31), 1) + array_fill_keys(range(45, 63), 1));
```

Comment:
You could change the variable name to $warningList so it is clearer

Reply:
Good call. Changing.